staging/src/k8s.io/api/core/v1/register.go

```diff
@@ -14,18 +14,24 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
+// Code generated by main. DO NOT EDIT.
+
 package v1
 
 import (
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/apimachinery/pkg/runtime/schema"
 )
 
-// GroupName is the group name use in this package
-const GroupName = ""
+// GroupName specifies the group name used to register the objects.
+const GroupName = "core"
```

Comment:
this wrong. For our logical core group we use the empty string. Where does this value come from? Do we follow the GroupName tag in the source? We could use that to specify the right name. Compare https://github.com/kubernetes/kubernetes/blob/c5445d3c56e06ab366b9cca34bd69c5cc386ec47/staging/src/k8s.io/code-generator/cmd/client-gen/generators/generator_for_group.go#L87